data/aliases.js

```diff
@@ -201,8 +201,13 @@ exports.BattleAliases = {
 	"floetteeternalflower": "Floette-Eternal",
 	"ashgreninja": "Greninja-Ash",
 	"zydog": "Zygarde-10%",
+	"zydoge": "Zygarde-10%",
```

Comment:
Apparently it's a thing, even though I don't appreciate shit ones

[21:53:21] +Kris: zydoge is another alias
[21:53:47] @Ridaz: really
[21:53:52] +Kris: yes really
[21:55:05] @Ridaz: tfw shit meme will never end